Assets/Scripts/Audio/AudioManager/MechanicAudioManager.cs

```diff
@@ -5,12 +5,15 @@ public class MechanicAudioManager : AbstractAudioManager {
 
     //Audioclips
     public AudioClip buttonClip;
+	public AudioClip unlockDoorClip;
```

Comment:
If these variables are used strictly within this class, consider making them private